src/core/Bridge.ts

```diff
@@ -1,9 +1,11 @@
 import React, { createElement, useMemo } from 'react'
 import { BridgeProvider, useBridgeValue } from 'use-context-selector'
 
-import { StateContext, ActionsContext } from './Provider'
+import { getContexts } from './Contexts'
+import { Scope } from './types'
 
-export const useBridge = () => {
+export const useBridge = (scope?: Scope) => {
```

Comment:
Another option is an option object.
```js
const useBridge = (opts?: { scope?: Scope }) => {
```

Reply:
https://github.com/pmndrs/jotai/pull/195#discussion_r523709569
As we don't allow `object` for scope for now, this should be fine, even if we changed our mind to support option object.